### PR TITLE
GEODE-10131: Remove set but unused variables

### PR DESCRIPTION
**Pull request by moleske**
- On AppleClang 13.1.6.13160021, the warning -Wunused-but-set-variable
  is appears as an error.  This allows compilation on AppleClang on
  MacOS 12.3

Authored-by: M. Oleske <michael@oleske.engineer>

### CODE REVIEW

**Review by pivotal-jbarrett — APPROVED** [review]
Well that was silly.

**Comment by pdxcodemonkey** [comment]
Looks like `CqPlusAuthInitializeTest.reAuthenticateWithDurable` is having trouble for some reason.  Give it a try on your local machine and see if it misbehaves for you.

**Comment by moleske** [comment]
Got the same error with geode hash of [199132ed71ac2e0324d530ca683e5870171093e2](https://github.com/apache/geode/tree/199132ed71ac2e0324d530ca683e5870171093e2)

Will have to take some time to see if this is because something on geode changed, as I'm very suspicious that deleting these two lines would have broken the test

**Comment by moleske** [comment]
@pdxcodemonkey - I have narrowed it down.  This [geode commit (hash 2554f42b925f2b9b8ca7eee14c7a887436b1d9db)](https://github.com/apache/geode/commit/2554f42b925f2b9b8ca7eee14c7a887436b1d9db) is the one that broke geode native.  Don't know if @jinmeiliao has any thoughts as the author of geode commit.  I haven't looked to see why it broke geode native, but feel very confident in saying this PR is not the cause of the breakage.  It also looks like https://github.com/apache/geode-native/pull/948 is failing for the same reason, so CI will be broken til this is resolved

**Comment by moleske** [comment]
being reverted [here](https://github.com/apache/geode/pull/7470), can rerun the pipelines after the change is merged in geode and a build is spat out of the geode pipeline for geode native pipeline to pickup

**Comment by moleske** [comment]
closing and opening to retrigger checks now that I think we have a good geode build

**Comment by moleske** [comment]
hmm that didn't work, will push an empty commit or rebase and force push, then complain about it

**Comment by moleske** [comment]
lgtm is failing cause now geode 1.14.3 and there is none (1.14.4 now).  #948 needs to be merged first

**Comment by moleske** [comment]
@pdxcodemonkey I think the ubuntu fixes that were made need to be copied to LGTM as it seems to be failing for the same reason over there

**Pull request merged**
